enforcer_synced.go

```diff
@@ -116,6 +116,22 @@ func (e *SyncedEnforcer) LoadPolicy() error {
 	return e.Enforcer.LoadPolicy()
 }
 
+// reloadPolicy reloads the policy from file/database, but only locks when replacing the model
+func (e *SyncedEnforcer) reloadPolicy() error {
```

Comment:
Modify LoadPolicy directly. No new function

Reply:
I modify this so a regular LoadPolicy is still locking the enforcer to prevent concurrent access on startup like today. Happy to modify this, if this is not a concern.

Reply:
I think it doesn't matter